index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,500;1,600;1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <!-- Nav Bar menu on mobile -->
+  <header>
+    <nav class="mobile-nav-bar">
+      <h1>My Logo</h1>
```

Comment:
- [ ] Kindly note that it's best to have just one `h1` tag on your page to indicate that to be the main heading of the entire page.  Using only one `h1` creates a `hierarchy` and gives your page a more `logical` structure. It makes it easy for readers (and those using `screen readers`) to skim and understand what your page is about. Each page should have only one main topic and headline. And that should be your only `h1`.

- [ ] Kindly change the `h1` and use `a` tag. Use `a` tag because it's a best practice to always make sure the logo one-stop ✋ link that users can click on to return back to the home page of your web application. This is also for convention, semantics, and user experience(UX). You can read more about it [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)